examples/app1.py

```diff
@@ -0,0 +1,92 @@
+"""
+Example application for simple usage of `flask-reuploaded` extension. In this
+example:
+- We upload images.
+- serve them manually.
+- Show, How to use database
```

Comment:
mixed upper and lower case, mixed dots at the end... please make this canonical looking